tests/search/test_LocalSearch.py

```diff
@@ -195,8 +192,6 @@ def test_prize_collecting(prize_collecting):
     improved = ls.search(sol, cost_evaluator)
     improved_cost = cost_evaluator.penalised_cost(improved)
 
-    # ...but an optimised prize-collecting solution is likely not complete.
-    assert_(improved.num_clients() < sol.num_clients())
     assert_(improved_cost < sol_cost)
```

Comment:
```suggestion
    assert_(improved.num_clients() < prize_collecting.num_clients)
    assert_(improved_cost < sol_cost)
```
Can we add one assert like this to check it's not complete?